### ISSUE TITLE
Different HRR and fire behaviour with different versions of FDS

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#920]
```
Please complete the following lines...

Application Version: 5.1.0 to 5.4.1 Serial
SVN Revision Number:
Compile Date:
Operating System: Win XP

Describe details of the issue below:

Using the same input file we obtained completely different simulation 
results by FDS versions 5.1.0 , 5.3.0 and 5.4.1. The version 5.1.0 gave
much more intensive fire, the version 5.4.1 much less intensive (see the 
files attached).

The case is a simple flat with very flamable furniture. Similar behaviour
was noticed also in another case.

We wonder if there were such substantial changes in the FDS code. Could 
you explain the significant difference for us, please?

Regards

Peter
```

Original issue reported on code.google.com by `upsyweis@savba.sk` on 2009-11-23 16:51:41

<hr>
- _Attachment: [case.zip](https://storage.googleapis.com/google-code-attachments/fds-smv/issue-946/comment-0/case.zip)_

**Comment by gforney** [comment]
```
Between 5.3 and 5.4 we changed the definition of REFERENCE_TEMPERATURE. This is a 
very important parameter, because it indicates at what temperature the material 
pyrolyzes. Your values of 110 C and 120 C seem far too low. I do not know of 
furniture materials that would burn at such low temperatures. Your HEAT_OF_REACTION

values are also unusual. 300 kJ/kg is very low, meaning that the reaction is going

to occur at a low temperature and it is going to require little additional heat to

make it occur. 

Any pyrolysis model, not just FDS, is going to be very sensitive to these 
parameters. In other words, small changes in FDS that occur with each minor release

might have significant changes on your results due to the sensitivity of these 
parameters. 

I suggest you verify the accuracy of your material properties by simulating a small

scale experiment, like a cone calorimeter test. Otherwise, you can have no 
confidence in your full-scale results.

&MATL ID            ='drevo'
      CONDUCTIVITY=0.1
      SPECIFIC_HEAT=6.0
      DENSITY=100.
      N_REACTIONS=1
      NU_FUEL=1.
      HEAT_OF_COMBUSTION=11000.
      HEAT_OF_REACTION=1500.
      REFERENCE_TEMPERATURE=110./

&MATL ID            ='drevo2'
      CONDUCTIVITY=0.1
      SPECIFIC_HEAT=1.0
      DENSITY=100.
      N_REACTIONS=1
      NU_FUEL=1.
      HEAT_OF_COMBUSTION=8000.
      HEAT_OF_REACTION=300.
      REFERENCE_TEMPERATURE=120./

```

Original issue reported on code.google.com by `mcgratta` on 2009-11-23 17:24:36

**Comment by gforney** [comment]
```
There is a wiki page containing release notes for all versions of FDS:

http://code.google.com/p/fds-smv/wiki/FDS_Release_Notes



```

Original issue reported on code.google.com by `drjfloyd` on 2009-11-23 17:56:34

**Comment by gforney** [comment]
```
I performed another simplified simulations (the kitchen only) with more realistic
material properties

&MATL ID            ='drevo'
      CONDUCTIVITY=0.2
      SPECIFIC_HEAT=1.5
      DENSITY=1000.
      N_REACTIONS=1
      NU_FUEL=1.
      HEAT_OF_COMBUSTION=25000.
      HEAT_OF_REACTION=1500.
      REFERENCE_TEMPERATURE=... ./


Using version 5.1.0, sustainable burning of the shelf was achieved for
REFERENCE_TEMPERATURE=330 and lower (after initialization fire finished)
Using version 5.4.2, sustainable burning of the shelf was not possible for
REFERENCE_TEMPERATURE=230 and higher.

In other words, we have to use ref. temperatures with a big difference about 100K to
obtain similar fire behaviour in FDS ver. 5.1.0 and FDS ver. 5.4.2. Was the pyrolysis
model changes (or ref. temperature definition change) between this versions so
significant, that they can explain such a difference ?
```

Original issue reported on code.google.com by `upsyweis@savba.sk` on 2009-11-27 14:11:42

**Comment by gforney** [comment]
```
There are many parameters that affect pyrolysis and flame spread. I suggest you do 
the following. Find the test case charring_solid.fds that is in the Pyrolysis folder

of the standard FDS installation:

C:\Program Files\FDS\FDS5\Examples\Pyrolysis

Replace the MATL and SURF lines with yours, and run the case with the old and new 
versions of FDS. This will help eliminate the gas phase parameters and allow us to

focus on changes made to the solid phase. 
```

Original issue reported on code.google.com by `mcgratta` on 2009-11-27 14:31:24

**Comment by gforney** [comment]
```
Have you tried what I have suggested?
```

Original issue reported on code.google.com by `mcgratta` on 2009-12-21 18:58:15

**Comment by gforney** [comment]
```
Yes, I have tried

a) for original SURF and MATL there were no significant difference between both FDS
versions

b) when I replaced them with SURF and MATL I used in my case, the temperature stayed
constant 20 in charring_solid_devc.csv file. I do not understand why is that. Since
then, I did not have much time to solve it.

The values are as follows:

&MATL ID            ='drevo'
      CONDUCTIVITY=0.2
      SPECIFIC_HEAT=1.5
      DENSITY=1000.
      N_REACTIONS=1
      NU_FUEL=1.
      HEAT_OF_COMBUSTION=25000.
      HEAT_OF_REACTION=1500.
      REFERENCE_TEMPERATURE=330./

&SURF ID='VZNIET'
      MATL_ID   = 'drevo'
      THICKNESS=0.03/
```

Original issue reported on code.google.com by `upsyweis@savba.sk` on 2009-12-22 10:57:20

**Comment by gforney** [comment]
```
For this special input file, you need to add the parameter EXTERNAL_FLUX to the SURF

line. This adds a specified heat flux that heats up your sample. This is a simple 
way to replicate a device like a cone calorimeter. Remember to remove this parameter

when you use the MATL and SURF lines in your full-scale simulation.
```

Original issue reported on code.google.com by `mcgratta` on 2009-12-22 14:25:40

**Comment by gforney** [comment]
```
I performed simulations for both FDS versions according to your unstructions, using
my material data. There is a difference in the results

Temperature after 600s:
ver. 5.1 : T=342
ver. 5.4 : T=394

I do not understand, where the problem is. Are my input data so strange, or non-typical
?
```

Original issue reported on code.google.com by `upsyweis@savba.sk` on 2010-01-14 13:58:21

**Comment by gforney** [comment]
```
Starting with FDS 5.3, we changed the definition of REFERENCE_TEMPERATURE to be more

consistent with the pyrolysis literature. Changes in minor release (5.1 to 5.2, for

example) have changes in some of the input definitions. How did you derive the 
REFERENCE_TEMPERATURE of 330?
```

Original issue reported on code.google.com by `mcgratta` on 2010-01-14 14:33:05

**Comment by gforney** [comment]
```
Originally, the problem arose with the temperature 350 - 400, and the data for that
material were provided by our collaborators, based primarilly on table values. We
used also T=330, which is based on the normalized value provided by some state
institution for ignition temperature and we used it as the first aproximation (as the
reference temperature itself we cannot determine).

But, my opinion was, that this problem is not caused by some concrete value, but by
something which is maybe connected with the pyrolisis model or the way we use it. We
observed these differences for larger range of values.
```

Original issue reported on code.google.com by `upsyweis@savba.sk` on 2010-01-14 14:57:30

**Comment by gforney** [comment]
```
I would guess that the difference is due to the definition of REFERENCE_TEMPERATURE.

The effect of this change would be to shift the 330 value 10 or 20 degrees higher or

lower. Keep in mind that REFERENCE_TEMPERATURE is not the same as an ignition 
temperature. There is no standard test that determines it.
```

Original issue reported on code.google.com by `mcgratta` on 2010-01-14 15:10:33

**Comment by gforney** [comment]
```
(No text was entered with this change)
```

Original issue reported on code.google.com by `mcgratta` on 2010-01-28 22:37:14
